bestie/server/test_result.go

```diff
@@ -57,6 +57,20 @@ func identifyStream(bazelBuildEvent bes.BuildEvent, streamId *build.StreamId) *b
 	return &stream
 }
 
+// Get the file size.
+func isValidFileSize(fileName string) error {
+	f, err := os.Stat(fileName)
```

Comment:
bytestream URLs aren't files, so these won't be found on-disk. I think a different approach is needed

Reply:
OK, see second commit.